.github/workflows/tox.yml

```diff
@@ -0,0 +1,33 @@
+name: "Tox Tests"
+
+on:
+  push:
+    branches-ignore:
+      - main
+
+jobs:
+  test:
+    name: Tox test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Tox tests
+        id: test
+        uses: fedora-python/tox-github-action@master
+        with:
+          tox_env: ${{ matrix.tox_env }}
+    strategy:
+      matrix:
+        tox_env:
```

Comment:
Do we have to specify the `-django{version}` of each `tox_env`? I see that [in `django-freeipa-auth`](https://github.com/enervee/django-freeipa-auth/blob/main/.github/workflows/tox.yml) we only specify the `py{version}` part.

Reply:
I could maybe mess around with it, but in my tox.ini I've got the envlist:
```
envlist = {py36,py37,py38,py39}-django22,{py36,py37,py38,py39,py310}-django32,{py38,py39,py310}-django40
```

So I ran `tox -l` in the terminal and got the list:
```
py36-django22
py37-django22
py38-django22
py39-django22
py36-django32
py37-django32
py38-django32
py39-django32
py310-django32
py38-django40
py39-django40
py310-django40
```

And I just slapped that in here and called it a day? I think if I didn't do them separately like this, they would maybe get grouped together in the jobs. Like here, I've got 12 jobs, but maybe if I just called out the python versions, I'd only have 5 jobs and some of them would run multiple Django versions?

No clue how tox_env thingy works, maybe grouping them up would be better than running them all simultaneously.

Reply:
Oh interesting! Ok, maybe that's an update we can make to the github job in the `django-freeipa-auth` repo as well, that way we can separate those tests into separate jobs 👍